examples/tts/conf/univnet/univnet.yaml

```diff
@@ -0,0 +1,84 @@
+name: "UnivNet"
+train_dataset: ???
+validation_datasets: ???
+
+defaults:
+  - model/generator: c16
+  - model/train_ds: train_ds
+  - model/validation_ds: val_ds
+
+model:
+  discriminator:
+    mpd:
+      periods: [2,3,5,7,11]
+      kernel_size: 5
+      stride: 3
+      use_spectral_norm: false
+      lrelu_slope: 0.2
+    mrd:
+      resolutions: [[1024, 120, 600], [2048, 240, 1200], [512, 50, 240]] # (filter_length, hop_length, win_length)
+      use_spectral_norm: false
+      lrelu_slope: 0.2
+  preprocessor:
+    _target_: nemo.collections.asr.parts.preprocessing.features.FilterbankFeatures
+    dither: 0.0
+    frame_splicing: 1
+    nfilt: 100
+    highfreq: 12000
+    log: true
+    log_zero_guard_type: clamp
+    log_zero_guard_value: 1e-05
+    lowfreq: 0
+    mag_power: 1.0
+    n_fft: 1024
+    n_window_size: 1024
+    n_window_stride: 256
+    normalize: null
+    pad_to: 0
+    pad_value: -11.52
+    preemph: null
+    sample_rate: 24000
```

Comment:
@blisc, should we specify sample rate in the name of file if it is not default? In my opinion, default sample rate for ASR is 16000, and for TTS is 22050.

Reply:
renamed to `univnet_24000` following the non-standard sr naming with [235f87b](https://github.com/NVIDIA/NeMo/pull/3186/commits/235f87ba67a76dc0bda24aeff4ed61568fbfc4f3)